solutions/security/ai/ai-assistant.md

```diff
@@ -26,13 +26,12 @@ The Elastic AI Assistant is designed to enhance your analysis with smart dialogu
 ::::
 
 ::::{admonition} Requirements
-* {{stack}} users: {{stack}} version 8.8.1 or later. Also note the Generative AI connector was renamed to OpenAI connector in 8.11.0.
-* {{stack}} users: an [Enterprise subscription](https://www.elastic.co/pricing).
-* {{serverless-short}} users: a [Security Analytics Complete subscription](/deploy-manage/deploy/elastic-cloud/project-settings.md).
+* {applies_to}`stack: ga` An [Enterprise subscription](https://www.elastic.co/pricing).
+* {applies_to}`serverless: ga` A [Security Analytics Complete subscription](/deploy-manage/deploy/elastic-cloud/project-settings.md).
 * To use AI Assistant, the **Elastic AI Assistant : All** and **Actions and Connectors : Read** [privileges](/deploy-manage/users-roles/cluster-or-deployment-auth/kibana-privileges.md).
 * To set up AI Assistant, the **Actions and Connectors : All** [privilege](/deploy-manage/users-roles/cluster-or-deployment-auth/kibana-privileges.md).
 * A [generative AI connector](/solutions/security/ai/set-up-connectors-for-large-language-models-llm.md), which AI Assistant uses to generate responses.
-
+* An [inference endpoint](/explore-analyze/machine-learning/nlp/ml-nlp-elser.md) and [Machine Learning node](/explore-analyze/machine-learning/setting-up-machine-learning.md).
```

Comment:
Couple notes here:

The default inference endpoints provided are what are used out of the box (either `.elser-2-elasticsearch` as you linked or `.multilingual-e5-small-elasticsearch` coming up in 9.2) , so while necessary, they don't need to manually create one. The ML node being available (with enough memory) is sufficient.

And WRT the ML Node, this is implicit on serverless. ML Nodes + autoscaling are enabled by default, so they don't need to do anything other than be on the Security Complete/EASE tier.